cvat/apps/engine/cloud_provider.py

```diff
@@ -808,7 +808,7 @@ def supported_actions(self):
         return allowed_actions
 
 
-class AzureBlobContainer(_CloudStorage):
+class AbsCloudStorage(_CloudStorage):
```

Comment:
I'm not sure Abs can be considered the established abbreviation for this service. Have you seen it anywhere? I'd prefer calling it `Azure...`.

Reply:
> I'm not sure Abs can be considered the established abbreviation for this service. Have you seen it anywhere?

It's admittedly not hugely prominent, but [examples exist](https://www.google.com/search?q=%22azure%20blob%20storage%22%20%22abs%22%20site%3Amicrosoft.com&client=ubuntu-sn&channel=fs&start=0).

> I'd prefer calling it `Azure...`.

IMO, just "Azure" is not specific enough. Would "AzureBlobCloudStorage" work for you?